cds_ils/migrator/relations/api.py

```diff
@@ -200,6 +201,23 @@ def link_records_and_serial(record_cls, search):
                     serial, record, SERIAL_RELATION, volume
                 )
 
+    def link_record_and_journal(record_cls, search):
```

Comment:
let's add test for the linking?

Reply:
I added them [here](https://github.com/CERNDocumentServer/cds-ils/blob/ac9f41bd3b0b24c000bf091d24e949ef5677b91d/tests/migrator/test_relations.py)